v3/src/components/graph/adornments/adornment-component-info.ts

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+
+export interface IAdornmentComponentInfo {
+  adornmentEltClass: string
+  Component: React.ComponentType<any>
+  inspectorList: {
+    position: number
+    label: string
+    hint: string
+  }
+  type: string
+}
+
+const gAdornmentComponentInfoMap = new Map<string, IAdornmentComponentInfo>()
+
+export function registerAdornmentComponentInfo(info: IAdornmentComponentInfo) {
+  gAdornmentComponentInfoMap.set(info.type, info)
+}
+
+export function getAdornmentComponentKeys() {
+  return Array.from(gAdornmentComponentInfoMap.keys())
+}
+
+// undefined is supported so callers do not need to check the id before passing it in
```

Comment:
```suggestion
// undefined is supported so callers do not need to check the type before passing it in
```
no doubt a copy-paste error that originated in the source